server/src/main/java/org/uiautomation/ios/utils/SimulatorSettings.java

```diff
@@ -56,7 +60,7 @@
   private InstrumentsVersion instrumentsVersion;
 
   public SimulatorSettings(HostInfo info, String sdkVersion, boolean is64bit, DeviceType device,
-                           DeviceVariation variation) {
+                           DeviceVariation variation) throws NotActiveException {
```

Comment:
I'm not keen on the use of `NotActiveException` here. The java doc for `NotActiveException` is ;

> Thrown when serialization or deserialization is not active.

There is no stream here that we are serializing / deserializing. Can `SessionNotInitializedException ` be used instead?